catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -236,13 +246,12 @@
 		<argument ref="federatedSources"/>
 		<argument ref="resourceReaderSortedList"/>
 		<argument ref="sorted"/>
```

Comment:
Do you mean can't?  It was using the same pool previously, so I'm not sure we want to change it.  The pool we changed was the cacheThreadPool, not the queryThreadPool (we just changed the name from pool to queryThreadPool).